nbtlib/tag.py

```diff
@@ -42,6 +42,7 @@
 from struct import Struct, error as StructError
 import numpy as np
 
+import nbtlib

```

Comment:
I dont know.
Why should not you use `import .path` here?

Reply:
```python
# in path.py
from .tag import Int, String, List, Compound
```

The `path` module depends on tags, so if the `tag` module directly imports `path`, none of the tag classes would be defined by the time `path.py` gets executed. It's related to circular import problems.